samples/apps/copilot-chat-app/scripts/README.md

```diff
@@ -1,45 +1,85 @@
-# Copilot Chat Setup Scripts
-
-> The PowerShell scripts in this directory require [PowerShell Core 6 or higher](https://github.com/PowerShell/PowerShell#get-powershell).
+# Copilot Chat Setup Scripts (local deployment)
 
 ## Before You Begin
-To run these scripts, you will need the following:
-- *REACT_APP_CLIENT_ID*
-  - This is the client ID (also known as application ID) associated with your Azure Active Directory (AAD) application registration, which you can find in the Azure portal.
-- *MY_AZUREOPENAI_OR_OPENAI_KEY*
+To run Copilot Chat, you will need the following:
+- *Application ID*
+  - This is the Client ID (i.e., Application ID) associated with your Azure Active Directory (AAD) application registration, which you can find in the Azure portal.
+- *Tenant ID*
+  - This is the Tenant ID (i.e., Directory ID) associated with your AAD app registration.
+  [Learn more about possible values for this parameter](https://learn.microsoft.com/en-us/azure/active-directory/develop/msal-client-application-configuration#authority).
+- *Azure OpenAI or OpenAI API Key*
   - This is your API key for Azure OpenAI or OpenAI
-- An updated [`appsettings.json`](../webapi/appsettings.json) file. At a minimum, you must fill out the `Completion` and `Embedding` sections per the instructions in the comments.
-
-For more information on how to prepare this data, [see the full instructions for Copilot Chat](../README.md).
 
-## Install Requirements
+## 1. Configure your environment
 ### Windows
-Open a PowerShell window as an administrator, navigate to this directory, and run the following command:
+If you have not already, install PowerShell 6 or newer from https://learn.microsoft.com/powershell/scripting/install/installing-powershell.
+> You can run `$PSVersionTable.PSVersion` in PowerShell to verify the version.
```

Comment:
If we are dropping the requirement for Powershell 6, can probably remove this (or at least say it's encouraged but not required)